spk/python3/Makefile

```diff
@@ -10,7 +10,7 @@ DEPENDS += cross/setuptools cross/pip cross/wheel
 DEPENDS += cross/cffi cross/bcrypt
 # Cross-compiled wheels
 DEPENDS += cross/lxml cross/pycrypto cross/pycurl cross/pyyaml
-DEPENDS += cross/msgpack-python cross/attr cross/acl cross/lz4
```

Comment:
I do not understand why removing `lz4` from Python 3... I may impact other packages running over Python 3.
What is the reason / motivation ?